### ISSUE TITLE
Aimbot snapping after kill

### ISSUE THREAD

**Issue by Th3yS33M3** [designer1337/aristois-legit#51]
Any ideas on how to stop the aimbot snapping after every kill (single fire weapons)?
Even with kill delay maxed, after the timer runs out on it, it snaps to next enemy(if it is in FoV range).

This seems super obvious in practical use, on Rifles this feature is welcome but pistols and snipers it's just unlegit.

**Comment by forntorn** [comment]
I think there is a smoothness slider and a fov slider maybe turn one up and one down?

**Comment by Th3yS33M3** [comment]
Well without triggerbot, it makes the single fire weapons aimbot completely useless then...

Point is to actually have the smooth low on singles so u snap on the heads, so that's not an option.

**Comment by forntorn** [comment]
Dude, trust me. Every "legit" cheat is like this mostly and if you don't like it don't use. Just buy one if you think it'll be better. Turn FOV lower (duh) and turn off dynamic fov, turn up smoothness. Not that hard dude, figure it yourself.

**Comment by Th3yS33M3** [comment]
> Dude, trust me. Every "legit" cheat is like this mostly and if you don't like it don't use. Just buy one if you think it'll be better. Turn FOV lower (duh) and turn off dynamic fov, turn up smoothness. Not that hard dude, figure it yourself.

No, every "legit" cheat actually have aim-switch after a kill disabled on singles, yes I'm figuring it out myself and no I do not pay for another pastas from someone else, you're not helping on this one as u don't seem to understand the issue.

**Comment by forntorn** [comment]
I'm pretty sure I understand. Turn FOV down if you don't want it to go to the other enemy as fast and turn the smoothness up and turn off dyanmic fov. I'm trying to help you but you are highly uncooperative. 
After you kill the guy maybe you should just stop pressing mouse1 or whatever you got it set too. Either way, I doubt Designer1337 will have many different things to say as I.

**Comment by Th3yS33M3** [comment]
> I'm pretty sure I understand. Turn FOV down if you don't want it to go to the other enemy and turn the smoothness up and turn off dyanmic fov. I'm trying to help you but you are highly uncooperative.
> After you kill the guy maybe you should just stop pressing mouse1 or whatever you got it set too. Either way, I doubt Designer1337 will have many different things to say as I.

Tldr; I'm requesting for a feature to be added, "aim-switch" option on weapon settings, ready to pay for it.

**Comment by designer1337** [comment]
i added feature called "kill delay" yesterday, this is what u need prolly.

**Comment by Th3yS33M3** [comment]
> i added feature called "kill delay" yesterday, this is what u need prolly.

Thanks for that, sadly it still snaps after the kill delay timer has run out, that's why I'm asking for full removal of the aim after kill on singles :)

**Comment by MrClue** [comment]
So you want it to completely stop the aimbot after the first kill?

**Comment by Th3yS33M3** [comment]
Yes, until mouse1 is pressed again :)
